docs/roman/stpipe/user_pipeline.rst

```diff
@@ -154,39 +156,37 @@ equivalent of the parameter file above:
 
 .. code-block:: python
 
-    from stpipe.pipeline import Image2Pipeline
+    from romancal.pipeline import ExposurePipeline
 
     steps = {
-        'jump':{'rejection_threshold': 180.,
-                'three_group_rejection_threshold': 190.,
-                'four_group_rejection_threshold':195.
-    }
+        'source_catalog':{ 'aperture_ee1': 3, 'aperture_ee2': 5, 'aperture_ee3': 7}
+	}
 
     pipe = ExposurePipeline(steps=steps)
 
 Running a Pipeline
 ==================
 
-From the commandline
---------------------
+From the command line
+---------------------
 
-The same ``strun`` script used to run Steps from the commandline can
+The same ``strun`` script used to run Steps from the command line can
 also run Pipelines.
 
 The only wrinkle is that any parameters overridden from the
-commandline use dot notation to specify the parameter name.  For
-example, to override the ``rejection_threshold`` value on the ``jump``
+command line use dot notation to specify the parameter name.  For
+example, to override the ``aperture_ee1`` value on the ``source_catalog``
 step in the example above, one can do::
 
-    > strun romancal.pipeline.ExposurePipeline --steps.jump.rejection_threshold=180.
+    > strun romancal.pipeline.ExposurePipeline --steps.source_catalog.aperture_ee1=3
 
 From Python
 -----------
 
 Once the pipeline has been configured (as above) it can be executed
 using run.
 
-    pipe.run(input_data)
+    pipe(input_data)
```

Comment:
```suggestion
    pipe.run(input_data)
```
Using `pipe(input_data)` executes `pipe.__call__` (which is deprecated). Please restore the example from main `pipe.run`.